### PR TITLE
Add an interesting paper from Tomas et al. from 2005.

### PR DESCRIPTION
**Pull request by ltratt**

### CODE REVIEW

File: softdev.bib [hunk 1]
```diff
@@ -1591,3 +1591,11 @@ @Article{killick12optimal
     Url     = {http://dx.doi.org/10.1080/01621459.2012.737745},
     month   = Dec,
 }
+
+@inproceedings{kalibera05precision,
+    title = "Benchmark precision and random initial state",
+    author = "Kalibera, Tomas and Bulej, Lubomir and Tuma, Petr",
+    booktitle = "SPECTS",
```
Comment (snim2) [inline softdev.bib]: Should this be `booktitle = "{SPECTS}"`?

**Comment by snim2** [comment]
I notice that this file mixes keys in lower case with capitalised keys, and quotes with braces e.g.:

```
    Title   = {Optimal Detection of Changepoints With a Linear Computational Cost},
    Author  = {Rebecca Killick and Paul Fearnhead and Idris Eckley},
```

vs.

```
    title        = "{JSMeter}: Characterizing Real-World Behavior of {JavaScript} Programs",
    author       = "Paruj Ratanaworabhan and Benjamin Livshits and David Simmons and Benjamin Zorn",
```

Does this matter?

**Comment by ltratt** [comment]
BibTeX's format is a living nightmare, and we haven't fully homogenised everything. In general, as far as I know, it doesn't matter if values are quoted or curly-bracketed; but curly brackets *inside* are significant (they prevent BibTeX lower-casing everything).

File: softdev.bib (hunk 1 above)
Comment (ltratt) [inline softdev.bib]: I think the `booktitle` key is left unfiddled with, so it's OK (at least, we have many others in this format that do the right thing -- I told you the rules were a nightmare :/).

**Pull request merged**